### ISSUE TITLE
Most of the Sensors are showing NA

### ISSUE THREAD

**Issue by ebnerjoh** [uajqq/weewx-davishealthapi#3]
Hi,

I was able to get at least the "current" values working, so I tried to show the sensor values.

But only the WLL Uptime, the Link Uptime and Console Battery are showing Values:

```
WLL Uptime: 0 days, 1 hour, 35 minutes
WLL Signalquality: N/A
WLL RSSI: N/A
WLL Supercap Volt: N/A
WLL Solar Volt: N/A
WLL Tx Battery: N/A
WLL Tx Battery Flag: N/A
WLL Console Battery: 6133 mV
WLL Packet Streak: N/A
WLL Error Packets: N/A
WLL Resynchs: N/A
WLL Link Uptime: 0 days, 1 hour, 35 minutes

```

**Comment by ebnerjoh** [comment]
Hi @uajqq , any ideas?

**Comment by uajqq** [comment]
Sorry for delay. When you say the "current" values are working, you mean Weather values, right?

**Comment by uajqq** [comment]
This issue has been resolved—please see #2
